examples/SwissPaymentSlipFpdf_Red_Slip.php

```diff
@@ -52,15 +52,16 @@
 $paymentSlipData = new RedPaymentSlipData();
 
 // Fill the data container with your data
-$paymentSlipData->setBankData('Seldwyla Bank', '8021 Zuerich');
+$paymentSlipData->setBankData('Seldwyla Bank', '8021 Zürich');
```

Comment:
I actually used "ue" deliberately as it was used in the ESR examples from Die Post.